src/dataManager.h

```diff
@@ -37,22 +38,45 @@ struct ControlsJson {
 		p2(p2) {}
 };
 
+struct PickupData {
+	std::string name;
+	int frame;
+	float duration;
+
+	PickupData() {}
+	PickupData(std::string n, int f, float d) : name(n), frame(f), duration(d) {}
+};
+
+struct PickupJson {
```

Comment:
Yeap, I thought about that too but I think it would be neater for us to store data in each json file in an individual struct each. That struct will be a representation of what is in the json file.

It is also easier to change if new objects need to be added to `pickup.json`. Anybody using `dataManager.h` can just call `dataManager->getPickupJson()` to get the data in that json file.